db/metrics.go

```diff
@@ -190,7 +190,7 @@ func (db database) GetBountiesByDateRange(r PaymentDateRange, re *http.Request)
 	} else {
 		orderQuery = " ORDER BY " + sortBy + "" + "DESC"
 	}
-	if limit > 1 {
+	if limit > 0 {
```

Comment:
why is this changed?

Reply:
The default limit is 1, so any number lesser than 1 should not trigger the limit query.